config-example.env

```diff
@@ -45,13 +45,17 @@ SQLPAD_LDAP_AUTH_ENABLED="false"
 SQLPAD_LDAP_URL=""
 
 # Base LDAP DN to search for users in.
-SQLPAD_LDAP_BASE_DN=""
+SQLPAD_LDAP_SEARCH_BASE=""
 
 # The bind user will be used to lookup information about other LDAP users.
-SQLPAD_LDAP_USERNAME=""
+SQLPAD_LDAP_BIND_DN=""
 
 # The password to bind with for the lookup user.
-SQLPAD_LDAP_PASSWORD=""
+SQLPAD_LDAP_BIND_CREDENTIALS=""
```

Comment:
I like "Password" better since it's a password.

Reply:
OK. I have changed to `SQLPAD_LDAP_PASSWORD `.